app/models/pipeline_run.rb

```diff
@@ -421,6 +421,16 @@ def db_load_ercc_counts
     update(total_ercc_reads: total_ercc_reads)
   end
 
+  def coverage_viz_summary_s3_path
+    # TODO(mark): Replace mock data with real data once we are running the sample.
+    return "s3://idseq-samples-development/markazhang/samples_12452/generate_coverage_viz_test/3.5/coverage_viz_summary.json"
+  end
+
```

Comment:
these files could've gone in assets.idseq.net

Reply:
Seconded. James led an effort a couple months ago to remove all explicit s3 paths from our open source code because it is too revealing

Reply:
Okay, moved to s3://assets.idseq.net